.travis.yml

```diff
@@ -1,2 +1,32 @@
+language: generic
+dist: xenial
 sudo: false
-script: exit 0
+before_install:
+- gem install bundler
+install:
+- bundle install
+before_script:
+- bundle exec hatchet ci:setup
+script: bundle exec rspec test/spec/
```

Comment:
This config seems to assume only Hatchet tests will be run, which is true for now, but if are you planning on adding any others, this will need lots of changes.

As an example, the Node buildpack also has: shellcheck, unit tests, and docker tests: https://github.com/heroku/heroku-buildpack-nodejs/blob/master/.travis.yml#L5

Reply:
I know, but that's something I can change once it becomes necessary.